src/Button/Button.js

```diff
@@ -212,6 +216,10 @@ export type Props = {
    * If defined, an `a` element will be used as the root node.
    */
   href?: string,
+  /**
+   * If `true`, and `fab` is `true`, will use mini floating action button styling.
+   */
+  mini: boolean,
```

Comment:
Should be optional?

Reply:
Sorry, what should?

Reply:
```js
mini?: boolean
```

Reply:
Are most of the other props wrong too?

Reply:
I've gone with adding a default for conistency. Happy to change it up if needed.

Reply:
I was just going by the treatment of the `href` prop above, but I'm no Flow expert...

Reply:
Seems all over the place looking at other components. Some have neither `?` nor default props. At least this is internally consistent.